content/wasm-functions/quickstart.md

```diff
@@ -437,8 +442,9 @@ If the build fails, check:
 
 * Are you in the `hello-spin` directory?
 * Did you successfully [install TinyGo](#install-language-specific-tooling)?
-* Are your versions of Go and TinyGo up to date?  The Spin SDK needs TinyGo 0.27 or above.
+* Are your versions of Go and TinyGo up to date? The Spin SDK needs TinyGo 0.27 or above.
```

Comment:
@adamreese Is this version requirement still correct?  Spin docs says TinyGo 0.35 and Go 1.22.  https://spinframework.dev/v3/quickstart#build-your-application